test/e2e/scenarios/retina/retina_scenarios_test.go

```diff
@@ -79,8 +74,75 @@ func TestE2ERetinaMetrics(t *testing.T) {
 
 	job.AddScenario(drop.ValidateDropMetric())
 
-	// todo: handle multiple scenarios back to back
 	job.AddScenario(tcp.ValidateTCPMetrics())
 
+	job.AddScenario(dns.ValidateBasicDNSMetrics())
+
+	job.AddStep(&azure.DeleteResourceGroup{}, nil)
+}
+
+func TestE2ERetinaAdvanceMetric(t *testing.T) {
+	job := types.NewJob("Validate that all advance metrics are present from the prometheus endpoint")
+	runner := types.NewRunner(t, job)
+	defer runner.Run()
+
+	curuser, _ := user.Current()
+
+	testName := curuser.Username + common.NetObsRGtag + strconv.FormatInt(time.Now().Unix(), 10)
+	sub := os.Getenv("AZURE_SUBSCRIPTION_ID")
+	loc := os.Getenv("AZURE_LOCATION")
+	if loc == "" {
+		loc = "eastus"
+	}
+
+	job.AddStep(&azure.CreateResourceGroup{
+		SubscriptionID:    sub,
+		ResourceGroupName: testName,
+		Location:          loc,
+	}, nil)
+
+	job.AddStep(&azure.CreateVNet{
+		VnetName:         "testvnet",
+		VnetAddressSpace: "10.0.0.0/9",
+	}, nil)
+
+	job.AddStep(&azure.CreateSubnet{
+		SubnetName:         "testsubnet",
+		SubnetAddressSpace: "10.0.0.0/12",
+	}, nil)
+
+	job.AddStep(&azure.CreateNPMCluster{
```

Comment:
I don't want to create two cluster to test different metrics. This is a costly operation. I would move the infrastructure piece out (till L123), and post cluster creation
- install Retina in basic mode 
- run the scenarios
- upgrade to advanced
- run the scenarios